engine/Components/Sanitizer.py

```diff
@@ -1,24 +1,52 @@
-#todo changes statement to something that we can use
-#todo change to something that the system can use
-#todo add brackets
-#todo change connectives to one common set
-#todo change lower case letters to upper case
+# todo changes statement to something that we can use
+# todo change to something that the system can use
+# todo add brackets
+# todo Change connectives to one common set
+# todo Change lower case letters to upper case
 
 
 # for example: change & to ^, | to v
 
 def add_brackets(statement):
```

Comment:
What does this function do?

Reply:
This and sanitize_statements() are removed in the next PR.